docs/user/security/securing-communications/elasticsearch-mutual-tls.asciidoc

```diff
@@ -43,15 +52,14 @@ This entails generating a "server certificate" for {es} to use on the HTTP layer
 
 NOTE: This is not the same as the <<configuring-tls-browser-kib,server certificate>> that {kib} will present to web browsers.
 
-You may choose to generate a client certificate and private key using the {ref}/certutil.html[`elasticsearch-certutil`] tool. If you
-followed the {es} documentation for {ref}/configuring-tls.html#node-certificates[generating node certificates], then you likely have already
-set up a certificate authority (CA) to sign the {es} server certificate. You may choose to use the same CA to sign the {kib} client
-certificate. For example:
+You may choose to generate a client certificate and private key using the {ref}/certutil.html[`elasticsearch-certutil`] tool. If you followed the {es} documentation for {ref}/security-basic-setup.html#generate-certificates[generating the certificates authority], then you already have a certificate authority (CA) to sign
+the {es} server certificate. You may choose to use the same CA to sign the {kib}
+client certificate. For example:
```

Comment:
So, if users followed the enrollment flow like we assumed above, they wouldn't have generated a CA for Elasticsearch. In that case, they would need to generate a new CA _and_ configure Elasticsearch to trust that CA too.

So I think this whole section needs to be reworked, but we can leave it as-is for now and address it as a follow-on task for #118283.